dojo/templates/dojo/view_finding.html

```diff
@@ -273,13 +273,52 @@ <h3 class="pull-left to_highlight finding-title">
                             </a>
                         </td>
                     {% endif %}
-                    {% if finding.cve %}
+                    <!-- {% if finding.cve %}
+                        <td>
+                            <a target="_blank" href="https://cve.mitre.org/cgi-bin/cvename.cgi?name={{ finding.cve }}">
+                                <i class="fa fa-external-link"></i> {{ finding.cve }}
+                            </a>
+                        </td>
+                    {% endif %} -->
+
+                    <!-- {% if finding.cve.startswith('CVE') %}
                         <td>
                             <a target="_blank" href="https://cve.mitre.org/cgi-bin/cvename.cgi?name={{ finding.cve }}">
                                 <i class="fa fa-external-link"></i> {{ finding.cve }}
                             </a>
                         </td>
+                    {% endif %} -->
+
+                    {% if finding.cve|slice:":3" == 'CVE' %}
+                        <td>
+                            <a target="_blank" href="https://cve.mitre.org/cgi-bin/cvename.cgi?name={{ finding.cve }}">
+                                <i class="fa fa-external-link"></i> {{ finding.cve }}
+                            </a>
+                        </td>
+                    {% elif finding.cve|slice:":2" == 'WS' %}
+                    <td>
+                        <a target="_blank" href="https://URL/cgi-bin/cvename.cgi?name={{ finding.cve }}">
```

Comment:
To have an ineffective link, you can use `href="javascript:;"` instead.

Reply:
Instead of having a partial list of possible letters (you will miss some anyways), you may want to catch anything that is not `CVE` there.

Reply:
Yes, I fixed this on the latest version, because I was unable to find links to DB.